subsys/bluetooth/controller/flash_driver.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <errno.h>
+
+#include <zephyr.h>
+#include <device.h>
+#include <soc.h>
+#include <flash.h>
+
+#include "ble_controller_soc.h"
+
+#include <logging/log.h>
+#define LOG_MODULE_NAME blectrl_flash
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_DBG);
+
+/* TODO: Shouldn't this macro be defined in some kernel header? Doesn't seems
+ * like it is available globally. */
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
I was surprised to not find any common definition of these, shouldn't this be centralized or something? :thinking:

Reply:
There is one definition of `min` in misc/util.h

Reply:
:man_facepalming: apparently grepped without `-i`..